pkg/notifier/sink.go

```diff
@@ -0,0 +1,19 @@
+package notifier
+
+import (
+	"context"
+
+	"github.com/kubeshop/botkube/pkg/config"
+)
+
+// Sink sends event notifications to the sinks.
+type Sink interface {
+	// SendMessage sends a generic message for a given source bindings.
+	SendMessage(context.Context, any, []string) error
```

Comment:
Maybe it could stay as `SendEvent`, as we don't send message, but raw event data? What do you think?